docs/release_notes/v1.0.0.adoc

```diff
@@ -0,0 +1,158 @@
+https://download.elastic.co/downloads/ecctl/1.0.0/ecctl_1.0.0_linux_amd64.tar.gz[ecctl_1.0.0_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.0.0-whats-new"]
+==== What's new
+
+* *Deployment create command now supports flag-based actions.* Instance definition flags for the `ecctl deployment create` command are now available. Using a predefined deployment definition JSON file is still optional with the `--file` flag.
+
+* *Deployment show command can now generate payload to update a deployment.* With the addition of a new `--generate-update-payload` flag for the `ecctl deployment show` command, you can obtain a valid update payload from an existing deployment. This payload can be manually edited to be used in conjunction with the `ecctl deployment update` command.
+
+* *Enterprise Search support* All relevant commands now support Enterprise Search as a resource kind.
++
+[source,sh]
+--
+$ ecctl deployment plan cancel --kind enterprise_search
+--
+
+* *New Elasticsearch keystore commands.* The following commands have been added:
++
+--
+`ecctl deployment elasticsearch keystore show`:: Shows the settings from the Elasticsearch resource keystore.
+`ecctl deployment elasticsearch keystore update`:: Updates the contents of an Elasticsearch resource keystore.
+--
+
+* *New User-Agent header.* API calls made through ecctl now use `ecctl/<version>` instead of `cloud-sdk-go/<version>`.
+
+* *Command-layer testing* It is now possible to write tests for commands using the new `testutils.RunCmdAssertion()` testing function:
++
+[source,go]
+--
+func Test_showCmd(t *testing.T) {
+<...>
+        {
+        	name: "succeeds with JSON format",
+        	args: testutils.Args{
+        		Cmd: showCmd,
+        		Args: []string{
+        			"show", util.ValidClusterID, "--ref-id=main-elasticsearch",
+        		},
+        		Cfg: testutils.MockCfg{
+        			OutputFormat: "json",
+        			Responses: []mock.Response{
+        				mock.New200ResponseAssertion(
+        					reqAssertion, mock.NewByteBody(showRawResp),
+        				),
+        			},
+        		},
+        	},
+        	want: testutils.Assertion{
+        		Stdout: string(showJSONOutput) + "\n",
+        	},
+        },
+    <...>
+    for _, tt := range tests {
+    		t.Run(tt.name, func(t *testing.T) {
+    			testutils.RunCmdAssertion(t, tt.args, tt.want)
+    		})
+    	}
+--
+
+[float]
+[id="{p}-release-notes-v1.0.0-bug-fixes"]
+==== Bug fixes
+
+* *Inability to create or update deployments through Ansible.* ecctl should behave in the same manner when executed from the terminal or any automation tool. A bug which showed `os.Stdin` as always populated when coupled with Ansible has been fixed.
+
+* *Allocator vacate flag parsing has been fixed* The `--target` flag was being ignored due to using `GetStringSlice()` when the flag declaration is `Array`. Flag parsing for allocator vacate operations now works as documented.*
```

Comment:
Is the trailing `*` intended?

Reply:
Ah! you're right! thanks for catching that 🙇‍♀️